Samples/Server/NativeServersUnitTests/NativeServersUnitTests.vcxproj

```diff
@@ -59,14 +59,15 @@
       <PrecompiledHeader>Use</PrecompiledHeader>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <AdditionalIncludeDirectories>$(ProjectDir);Common;\inc;Content;$(VCInstallDir)UnitTest\include;..\..\..\Libraries\DXUT\Core;..\..\..\Libraries\DXUT\Optional;..\..\..\Libraries\DXUT\Remoting;..\..\..\Libraries\WebRTC\headers;..\..\..\Plugins\NativeServerPlugin\inc;..\..\..\Libraries\ConfigParser\inc;..\..\..\Libraries\WebRTC\headers\third_party\jsoncpp\source\include\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>WEBRTC_USE_OPENH264;_CRT_SECURE_NO_WARNINGS;_DEBUG;_WINDOWS;WEBRTC_WIN;_WIN32_WINNT=0x0600;WIN32_LEAN_AND_MEAN;NOMINMAX;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>$(ProjectDir);$(VCInstallDir)UnitTest\include;\inc;..\Directx-SpinningCube\Common;..\Directx-SpinningCube\Content;..\..\..\Plugins\NativeServerPlugin\inc;..\..\..\Libraries\ConfigParser\inc;..\..\..\Libraries\DXUT\Core;..\..\..\Libraries\DXUT\Optional;..\..\..\Libraries\DXUT\Remoting;..\..\..\Libraries\UserInterface\inc;..\..\..\Libraries\WebRTC\headers;..\..\..\Libraries\WebRTC\headers\third_party\jsoncpp\source\include\;..\..\..\Libraries\Freeglut\include\GL;..\..\..\Libraries\glext\include\GL;..\..\..\Libraries\Glew\include\GL;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;TEST_BUFFER_CAPTURER;_DEBUG;_WINDOWS;WEBRTC_WIN;NOMINMAX;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Is there a specific reason why you added a TEST_BUFFER_CAPTURER definition? Can we just include the friend class as default?

Reply:
I'd like to keep our capturer class as clean as possible so that flag makes the necessary declarations only visible to test projects. Anyway, I've refactored the code so that we don't need to modify <PreprocessorDefinitions> anymore and it also looks nicer.